### ISSUE TITLE
Hold Note Release Timings

### ISSUE THREAD

**Issue by Dragonmon101** [ShadowMario/FNF-PsychEngine#15750]
### What is your question?

Would it be possible to add in something like this with messing around with the code in PlayState.hx? (and its editor playtesting counterpart)

**Comment by KELOTOMU** [comment]
Yes, but also check the `objects/Note.hx` because that's where most of the note functionability is in.
